### PR TITLE
feat: Submit program load transactions in parallel

### PR DESCRIPTION
**Pull request by jackcmay**

### CODE REVIEW

File: src/loader.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ export class Loader {
     let userdata = Buffer.alloc(chunkSize + 16);
     let offset = 0;
     let array = data;
+    const transactions= [];
```
Comment (mvines) [inline src/loader.js]: nit: space between `s=`

File: src/loader.js [hunk 2]
```diff
@@ -66,11 +67,12 @@ export class Loader {
         programId: this.programId,
         userdata,
       });
-      await sendAndConfirmTransaction(this.connection, program, transaction);
+      transactions.push(await sendAndConfirmTransaction(this.connection, program, transaction));
```
Comment (mvines) [inline src/loader.js]: don't `await` here.   We want to simply collect all the `Promise`s into the transactions array here, not `await` for them to resolve.

Comment (jackcmay) [reply src/loader.js]: Good point!

**Comment by jackcmay** [comment]
@mvines Looks like token_program failed:

`FAIL test/token-program.test.js (9.145s)`

Please review and I'll squash and merge while trying to keep the sem version.

File: src/loader.js [hunk 3]
```diff
@@ -46,12 +46,12 @@ export class Loader {
...
 
     const chunkSize = 256;
-    let userdata = Buffer.alloc(chunkSize + 16);
     let offset = 0;
     let array = data;
+    const transactions = [];
     while (array.length > 0) {
       const bytes = array.slice(0, chunkSize);
-
+      let userdata = Buffer.alloc(chunkSize + 16);
```
Comment (mvines) [inline src/loader.js]: nit: const userdata = ...

Comment (mvines) [reply src/loader.js]: hey, why no const?

**Pull request merged**